src/Packagist/WebBundle/Resources/config/services.yml

```diff
@@ -1,7 +1,7 @@
 services:
     packagist.twig.extension:
         class: Packagist\WebBundle\Twig\PackagistExtension
-        arguments: [ @doctrine ]
+        arguments: [ '@doctrine' ]
```

Comment:
Not sure why you would mix using single `'` and double `"` quotes. I would prefer to see consistency in this file :-)